example/example_resourcemanager_test.go

```diff
@@ -0,0 +1,114 @@
+
+func CreateStack(ctx context.Context, provider common.ConfigurationProvider, client *resourcemanager.ResourceManagerClient) string {
+	stackName := fmt.Sprintf("test-%s", helpers.GetRandomString(8))
+	region, _ := provider.Region()
+	tenancy_ocid, _ := provider.TenancyOCID()
+
+	req := resourcemanager.CreateStackRequest{
+		CreateStackDetails: resourcemanager.CreateStackDetails{
+			CompartmentId: helpers.CompartmentID(),
+			ConfigSource: resourcemanager.CreateZipUploadConfigSourceDetails{
+				WorkingDirectory:     common.String("vcn"),
+				ZipFileBase64Encoded: common.String(zipFile),
+			},
+			DisplayName: common.String(stackName),
+			Description: common.String(fmt.Sprintf("%s-description", stackName)),
+			Variables: map[string]string{
+				"compartment_ocid": *helpers.CompartmentID(),
+				"region":           region,
+				"tenancy_ocid":     tenancy_ocid,
+			},
+		},
+	}
+
+	stackResp, err := client.CreateStack(ctx, req)
+	helpers.FatalIfError(err)
+
+	fmt.Println("create stack completed")
+	return *stackResp.Stack.Id
+}
+
+func UpdateStack(ctx context.Context, StackId string, client *resourcemanager.ResourceManagerClient) {
+	stackName := fmt.Sprintf("test-v1-%s", helpers.GetRandomString(8))
+
+	req := resourcemanager.UpdateStackRequest{
+		StackId: common.String(StackId),
+		UpdateStackDetails: resourcemanager.UpdateStackDetails{
+			DisplayName: common.String(stackName),
+			Description: common.String(fmt.Sprintf("%s-description", stackName)),
+		},
+	}
+
+	_, err := client.UpdateStack(ctx, req)
+	helpers.FatalIfError(err)
+
+	fmt.Println("update stack completed")
+}
+
+func ListStacks(ctx context.Context, client *resourcemanager.ResourceManagerClient) {
+	req := resourcemanager.ListStacksRequest{
+		CompartmentId: helpers.CompartmentID(),
+	}
+
+	_, err := client.ListStacks(ctx, req)
+	helpers.FatalIfError(err)
+
+	fmt.Println("list stacks completed")
+}
+
+func GetStack(ctx context.Context, StackId string, client *resourcemanager.ResourceManagerClient) {
```

Comment:
for the private methods, pls lower the case of first letter, otherwise, add document about public methods.

Reply:
change them to private method